main.go

```diff
@@ -3,13 +3,21 @@ package main
 import (
```

Comment:
please run `go fmt .`

Reply:
Oh, so you're suggesting I actually download Golang on my computer now?! 😞